src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("x is %2d, y is %4.2f, z is %15s" % (x, y, z))
```

Comment:
if you want to insert a string and you don't know how long it is, you can use %s. Also, if you want to include the quotations in the print, you can do the following:

`print("x is %d, y is %.2f, z is \"%s\"" % (x, y, z))`